bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/DataFlowDiagramConverter.java

```diff
@@ -0,0 +1,401 @@
+package org.dataflowanalysis.analysis.converter;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.HashMap;
+import java.util.List;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.log4j.Logger;
+import org.dataflowanalysis.analysis.converter.webdfd.*;
+import org.dataflowanalysis.dfd.datadictionary.*;
+import org.dataflowanalysis.dfd.dataflowdiagram.*;
+import org.dataflowanalysis.dfd.dataflowdiagram.Process;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class DataFlowDiagramConverter extends Converter {
+
+    private final Map<Pin, String> mapInputPinToFlowName = new HashMap<>();
```

Comment:
try to keep the naming constant. Put the type info at the end (inputPinToFlowNameMap) as you have done for other attributes and variables.